web/dispers/dispers.go

```diff
@@ -33,16 +33,19 @@ func createConcept(c echo.Context) error {
 
 func deleteConcept(c echo.Context) error {
 	concept := c.Param("concept")
-	err := enclave.DeleteConcept(concept)
+
+	err := enclave.DeleteConcept([]byte(concept))
```

Comment:
Why the `[]byte` cast?

Reply:
I planned to put every encrypted data as `[]byte`. It's just a kind of convention because encryption/decryption are just fake for now.